tests/test_internet_iface_sel.py

```diff
@@ -0,0 +1,73 @@
+import sys
```

Comment:
sorry I'm not sure I understand what you say. Do you mean add in the file `test_interfaces.py` you have added?